webclient/app/src/app/commons/appTemplateSelectCard/AppTemplateSelectCard.jsx

```diff
@@ -18,27 +16,18 @@ function AppTemplateSelectCard(props) {
     }
 
     return (
-        <Card color={"error"}>
-            <CardMedia
-                component="img"
-                height="150rem"
-                image={template.image}
-                alt="green iguana"
-                className={appTemplateSelectCardStyles.cardMedia}
-            />
-            {renderSelectedAlert()}
-            <CardContent>
+        <Card elevation={5}>
+            <CardActionArea onClick={() => onSelection(template)} disabled={selected}>
+            <CardContent >
                 <Typography gutterBottom variant="h5" component="div">
                     {template.name}
                 </Typography>
                 <Typography variant="body2" color="text.secondary">
                     {template.description}
                 </Typography>
             </CardContent>
-            <CardActions className={appTemplateSelectCardStyles.actionsWrapper}>
```

Comment:
It could be confusing for the user to not see a select button at the end of the card